tests/kafkatest/services/console_consumer.py

```diff
@@ -210,7 +210,10 @@ def start_cmd(self, node):
 
         # LoggingMessageFormatter was introduced after 0.9
         if node.version > LATEST_0_9:
-            cmd += " --formatter org.apache.kafka.tools.consumer.LoggingMessageFormatter"
+            if node.version > LATEST_3_7:
+                cmd += " --formatter org.apache.kafka.tools.consumer.LoggingMessageFormatter"
+            else:
+                cmd += " --formatter kafka.tools.LoggingMessageFormatter"
```

Comment:
`LoggingMessageFormatter`,  `DefaultMessageFormatter`, `NoOpMessageFormatter` are not part of the public API and as far as I can tell the class names are not visible anywhere so this shouldn't be considered an API change.